app/assets/stylesheets/oregon_digital/_search.scss

```diff
@@ -4,4 +4,9 @@
 
 .my-shelf-button {
   margin-top: 15px;
+}
+
+.shift-right {
```

Comment:
This is problematic with long titles
![image](https://user-images.githubusercontent.com/11052958/63385487-e994f280-c355-11e9-84b6-07f91accb720.png)

Maybe we can work it into the resource-type div and simplify it to a `class="pull-right"`?

Reply:
I tried pull right but it didnt seem to operate as expected.

Reply:
pushed 7b547ab as a fix for works with long names

![image](https://user-images.githubusercontent.com/11052958/63390004-72b22680-c362-11e9-82ff-46d07a696d94.png)